demo-shell/src/app/components/trashcan/trashcan.component.scss

```diff
@@ -0,0 +1,5 @@
+@media screen and (-ms-high-contrast: active), screen and (-ms-high-contrast: none) {
+    .adf-trashcan-center-delete-column-ie {
```

Comment:
let's remove this from demo shell and try to have some clean implementation that works out of the box